pages/writing/writing.md

```diff
@@ -632,6 +642,9 @@ In the debugging pane of the Julia extension, click `Run and Debug` to start the
 The program will automatically halt when it hits a breakpoint.
 Using the toolbar at the top of the editor, you can then _continue_, _step over_, _step into_ and _step out_ of your code.
 The debugger will open a pane showing information about the code such as local variables inside of the current function, their current values and the full call stack.
+The debugger can be [sped up](https://www.julia-vscode.org/docs/dev/userguide/debugging/#Settings-to-speed-up-the-debugger) by selectively compiling modules that you will not need to step into via the `+` symbol at the bottom of the debugging pane.
+It is often easiest to start by adding `ALL_MODULES_EXCEPT_MAIN` to the compiled list, and then selectively remove the modules you need to have interpreted.
```

Comment:
In my experience, the VSCode debugger can be prohibitively and frustratingly slow without some compilation as soon as you move away from toy code.

Reply:
Good point!